plugins/cost-insights/src/client.ts

```diff
@@ -125,170 +126,18 @@ export class ExampleCostInsightsClient implements CostInsightsApi {
     return projectDailyCost;
   }
 
-  async getProductInsights(
-    productInsightsOptions: ProductInsightsOptions,
-  ): Promise<Entity> {
-    const projectProductInsights = await this.request(productInsightsOptions, {
-      id: productInsightsOptions.product,
-      aggregation: [80_000, 110_000],
-      change: {
-        ratio: 0.375,
-        amount: 30_000,
-      },
-      entities: [
-        {
-          id: null, // entities with null ids will be appear as "Unlabeled" in product panels
-          aggregation: [45_000, 50_000],
-          change: {
-            ratio: 0.111,
-            amount: 5_000,
-          },
-          entities: [],
-        },
-        {
-          id: 'entity-a',
-          aggregation: [15_000, 20_000],
-          change: {
-            ratio: 0.333,
-            amount: 5_000,
-          },
-          entities: [],
-        },
-        {
-          id: 'entity-b',
-          aggregation: [20_000, 30_000],
-          change: {
-            ratio: 0.5,
-            amount: 10_000,
-          },
-          entities: [],
-        },
-        {
-          id: 'entity-c',
-          aggregation: [0, 10_000],
-          change: {
-            ratio: 10_000,
-            amount: 10_000,
-          },
-          entities: [],
-        },
-      ],
-    });
+  async getProductInsights(options: ProductInsightsOptions): Promise<Entity> {
+    const projectProductInsights = await this.request(
+      options,
```

Comment:
For the purposes of this PR, I scoped the request for mock data down to just productInsights requests so the sorting action can be visualized. Toggling a project will still return the same dataset.

Maybe we can address mocking client data more comprehensively in another PR? Right now product insights data is hardcoded in but it'd be preferable to have a programmatic way of doing it.

Reply:
@ryanvazquez yeah we can leave mock data for as a separate task. We might still want to just use one `productInsights` here since they're the same here

Reply:
Yep, agreed.